src/components/Proposal/View/ProposalCard.tsx

```diff
@@ -17,34 +20,83 @@ interface Props {
   proposal: ProposalAttributes
   highlight?: boolean
   isOverBudget?: boolean
+  showBudget?: boolean
+  isDisabled?: boolean
+  hideUser?: boolean
+  showLeadingVP?: boolean
+  hideEndDate?: boolean
 }
 
-export default function ProposalCard({ proposal, highlight, isOverBudget }: Props) {
+export default function ProposalCard({
+  proposal,
+  highlight,
+  isOverBudget,
+  showBudget,
+  isDisabled,
+  hideUser,
+  showLeadingVP,
+  hideEndDate,
+}: Props) {
   const t = useFormatMessage()
   const { id, title, user, start_at, finish_at } = proposal
   const { votes } = useProposalVotes(id)
   const dateText = useProposalDateText(start_at, finish_at)
+  const { formatNumber } = useIntl()
+  const budget = getBudget(proposal)
+  const { winningChoice, userChoice } = useWinningChoice(proposal)
 
   return (
     <Link
       className={classNames(
         'ProposalCard',
-        highlight && (isOverBudget ? 'ProposalCard--overbudget' : 'ProposalCard--highlight')
+        highlight && (isOverBudget ? 'ProposalCard--overbudget' : 'ProposalCard--highlight'),
+        isDisabled && 'ProposalCard--disabled'
       )}
-      href={locations.proposal(id)}
+      href={proposalUrl(id)}
+      target="_blank"
```

Comment:
I think our link component already handles this

Reply:
It's a "local link", we need to specify it.